docs/UI/Patch-Configuration.md

```diff
@@ -0,0 +1,20 @@
+# Patch Configuration
+
+## Overview
+The Patch Configure page is where you can configure your patch builds. It is accessible from the [My Patches page](./My-Patches.md) by clicking on a patch that has not yet been configured. The page is divided into several sections.
+
+### Build Variant Selector
+The left side panel contains the build variants that are available for the project. You can can select any combination of build variants using cmd+click and shift+click. If you have a task name that is reused across multiple build variants it will automatically be deduplicated in the task panel.
```

Comment:
```suggestion
The left side panel contains the build variants that are available for the project. You can can select any combination of build variants using <kbd>Cmd</kbd> + click and <kbd>Shift</kbd> + click. If you have a task name that is reused across multiple build variants it will automatically be deduplicated in the task panel.
```